docs/assignment1.md

```diff
@@ -1,62 +1,52 @@
+| --- | --- | --- |
+| F1 | API to Create/Read/Update/Delete snippets | Users should be able to easily access edit, create and delete snippets by calling the tool from the CLI using arguments. |
+| F2 | Snippet editor | Users should be able to edit and save snippets via a basic Vim-like CLI editor with visual/insert/replace modes. |
+| F3 | Tags | Users should be able to add tags to snippets by which will make finding snippets easier. This should include language and whether the snippet is "starred". |
+| F5 | Search/Filter | Users can search for their snippets by content, tags, description, title, language. Like snippo -s "language:java tags:tag1,tag2 searchterm". |
+| F6 | Configurable syntax highlighting | Users should have their code for a specific language with highlighting of syntax that is configurable via a file (that they create or that has been provided) that maps keywords to colors. |
+| F7 | Readme/help for the tool | Users should be able to view a manual explaining the basic commands and how to use the tool. |
+| F8 (optional) | Authentication | Users should be able to create and use accounts where their personal protected snippets are stored. |
+| F9 | Snippet metadata | Users should be able to view snippet metadata such as creation and last modification date, count of times edited or opened. |
+| F10 (optional) | Import/Export snippets | Users should be able to import and export via a json file to reuse on a different machine using Snippo. |
+| F11 (optional) | Integrated with Github Gist API | Users should be able to use Snippo in conjunction with Github Gists: [https://developer.github.com/v3/gists/](https://developer.github.com/v3/gists/) |
+| F12 | Basic interactive UI in shell | On launch the program should display available commands (e.g. edit, create, delete) and execute the chosen ones until closed.  |
 
-| ID  | Short name  | Quality attribute | Description  |
-|---|---|---|---|
-| QR1  | Commands sanity checks | Reliability  | When the player issues a command, the syntax of the command shall always get validated against the format specified in F2 |
-| QR2  | Extensible world | Maintainability  | The video game shall be easilty extendable in terms of levels, worlds, interaction points  |
-| QR3  | Instantaneous results | Responsiveness  | Once the scores of all soccer players are provided by the user, the results of the virtual matches shall be available within 1 second |
-| QR4  | ... | ... | ... |
+## Quality Requirements:
 
-Each quality requirement must be tagged with the corresponding quality attribute (see corresponding slides of the first lecture for knowing them).
+Author(s): Serghei Mihailov, Bjorn Keyser, Yael Goede, Milos Delgorge
 
-Maximum number of words for this section: 1000
+| **ID** | **Short name** | **Quality attribute** | **Description** |
+| --- | --- | --- | --- |
+| QR1 | Configurable language highlighting | Extensibility | It should be easy for the user to add a new code language. |
```

Comment:
"Shall" not "Should" otherwise it seems optional. Your requirement and explanation don't match exactly, the quality attribute doesn't match.